install.sh

```diff
@@ -3232,6 +3275,25 @@ execute_installation() {
         setup_shared_hook_repositories
         echo # For visual separation
     fi
+
+    finalize_installation || return 1
+}
+############################################################
+# Sets the install directory.
+#
+# Returns: None
+############################################################
+set_install_dir() {
+    # First check if we have already a install directory set
+    INSTALL_DIR=$(git config --global githooks.installDir)
```

Comment:
I would:
- first check if `$INSTALL_DIR` is empty and set it to `$HOME/.githooks` if it is and `return`
- check if the directory exists and return
- fall back to `$HOME/.githooks` otherwise